modules/aws-eks-managed-node-groups/managed-launch-templates.tf

```diff
@@ -34,9 +34,9 @@ resource "aws_launch_template" "managed_node_groups" {
   }
 
   metadata_options {
-    http_endpoint               = var.context.http_endpoint
-    http_tokens                 = var.context.http_tokens
-    http_put_response_hop_limit = var.context.http_put_response_hop_limit
+    http_endpoint               = try(var.context.http_endpoint, "enabled")
+    http_tokens                 = try(var.context.http_tokens, "required") #tfsec:ignore:aws-autoscaling-enforce-http-token-imds
```

Comment:
Is there a way we could not ignore this one?

Reply:
yes, it would be changing line 38 to 
```hcl
    http_tokens                 = "required"
```

However, I don't know the ramifications of this change on users and their applications

Reply:
For reference - https://github.com/terraform-aws-modules/terraform-aws-eks/issues/1749